controllers/idler/idler_controller.go

```diff
@@ -635,6 +631,19 @@ func nextPodToBeKilledAfter(log logr.Logger, idler *toolchainv1alpha1.Idler) *ti
 	return &d
 }
 
+// findShortestDuration finds the shortest duration the given durations
+func findShortestDuration(durations ...*time.Duration) *time.Duration {
```

Comment:
we expect that the function always returns some non-nil value, so let's return it as  non-pointer value, there is no need to keep the pointer in there.

Reply:
done in [bfc3cda](https://github.com/codeready-toolchain/member-operator/pull/630/commits/bfc3cdab10e2b06e72dbf287053a7f7188fad488)